perllib/FixMyStreet/Script/Inactive.pm

```diff
@@ -23,12 +23,63 @@ has cobrand => (
     },
 );
 
+has anonymous_user => (
+    is => 'lazy',
+    default => sub {
+        FixMyStreet::DB->resultset("User")->find_or_create({
+            email => 'removed-automatically@' . FixMyStreet->config('EMAIL_DOMAIN'),
+        });
+    }
+);
+
 sub users {
     my $self = shift;
     $self->anonymize_users;
     $self->email_inactive_users;
 }
 
+sub reports {
+    my $self = shift;
+
+    # Need to look though them all each time, in case any new updates/alerts
+    my $problems = FixMyStreet::DB->resultset("Problem")->search({
+        lastupdate => { '<', interval($self->anonymize) },
+        state => [ FixMyStreet::DB::Result::Problem->closed_states(), FixMyStreet::DB::Result::Problem->fixed_states() ],
+    });
+
+    while (my $problem = $problems->next) {
+        say "Anonymizing problem #" . $problem->id if $self->verbose;
+
+        # Remove personal data from the report
+        $problem->update({
+            user => $self->anonymous_user,
+            name => '',
+            anonymous => 1,
+            send_questionnaire => 0,
+        })
+            if $problem->user != $self->anonymous_user;
```

Comment:
This formatting is weird and a bit confusing in that the if does not look like it's associated with the update.